chemie/sugepodden/models.py

```diff
@@ -0,0 +1,49 @@
+from ckeditor.fields import RichTextField
+from django.contrib.auth.models import User
+from django.urls import reverse
+from django.db import models
+from sorl.thumbnail import ImageField
+from django.conf import settings
+from audiofield.fields import AudioField
+
+
+class Podcast(models.Model):
+    title = models.CharField(max_length=100)
+    content = RichTextField(verbose_name="beskrivelse", config_name="news")
+    published_date = models.DateTimeField(auto_now_add=True)
+    image = ImageField(upload_to="Images", verbose_name="Bilde")
```

Comment:
Lurt at det kanskje har et litt bedre navn en "Images". Feltet brukes til å se forskjell på hvilke bilder som hører til hvilke apps :p

Reply:
Scratch that, ser appen er veldig WIP